server/schema.ts

```diff
@@ -67,3 +67,17 @@ export const Attendee = mongoose.model<IAttendeeMongoose>("Attendee", new mongoo
 		required: true
 	}
 }));
+
+// Master list of available tags
+export interface ITagsList {
+	tags: string[];
```

Comment:
I think it would make more sense to have a collection of tags that look like `{name: "tag-name"}` right? That way we can add additional properties later if we want in addition to being able to use the DB to sort, find, etc.

Reply:
I agree, this is also what the graphQL schema expects.

Reply:
Yeah, good point. I'm changing this schema